css/style.css

```diff
@@ -1 +1,6 @@
-/* file for your css */
\ No newline at end of file
+.space {
+  min-width: 100px;
+}
+.red {
+  background-color: red;
+}
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }